Gemfile.lock

```diff
@@ -193,6 +229,9 @@ DEPENDENCIES
   annotate
   bootsnap
   debug
+  devise_token_auth (>= 1.2.0)!
+  letter_opener
+  omniauth
```

Comment:
Did you run bundle after removing omniauth from gemfile?

Reply:
no